docs/concepts/governance/bal-token.md

```diff
@@ -150,12 +149,9 @@ This results in the following emissions schedule:
     }
   }
 }
-```
-
 :::
 
-## Distribution
-
+Distribution
```

Comment:
This should probably keep a header format

Reply:
Correct.  Pushed changes.